projects/plugins/crm/src/automation/commons/triggers/events/class-event-deleted.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Jetpack CRM Automation Event_Deleted trigger.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Triggers;
+
+use Automattic\Jetpack\CRM\Automation\Base_Trigger;
+
+/**
+ * Adds the Event_Deleted class.
+ */
+class Event_Deleted extends Base_Trigger {
+
+	/**
+	 * Get the slug name of the trigger.
+	 * @return string
+	 */
+	public static function get_slug(): string {
+		return 'jpcrm/event_deleted';
+	}
+
+	/**
+	 * Get the title of the trigger.
+	 * @return string
+	 */
+	public static function get_title(): ?string {
```

Comment:
It's something we generally need to clean up in automations, but there's no chance this value is ever null, so it should just be `string`

```suggestion
	public static function get_title(): string {
```